tests/distributed_grids/get_boundary_ids_01.cc

```diff
@@ -0,0 +1,58 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2020 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+
+// create a tria mesh and copy it
```

Comment:
I don't think that is the proper description for this test. :wink:

Reply:
Please fix the description here and I am happy with it.

Reply:
I'll just do it myself in a follow-up pull request. Then we don't need to run CI again.